README

```diff
@@ -1,2 +1,10 @@
-# Neuromod: xxxx subdataset
+# CNeuromod: [xxxx subdataset](https://github.com/courtois-neuromod/xxxxx)
 
```

Comment:
I would add a general intro to the project:

The Courtois project on Neural Modelling ([Courtois NeuroMod](https://cneuromod.ca)) aims at training artificial neural networks using extensive experimental data on individual human brain activity and behaviour.